### ISSUE TITLE
Why there is a video in every fork

### ISSUE THREAD

**Issue by nitanagdeote** [vizhub-core/vizhub-feedback#782]
![image](https://github.com/vizhub-core/vzcode/assets/13540669/34f11e87-3207-4de3-98c2-23f957532650)

**Comment by curran** [comment]
Indeed, I'm thinking to make it not copy over the README when you fork. Currently it copies all the files including the README.
